web_src/js/components/DiffFileTree.vue

```diff
@@ -109,15 +110,20 @@ export default {
     updateVisibility(visible) {
       this.fileTreeIsVisible = visible;
       localStorage.setItem(LOCAL_STORAGE_KEY, this.fileTreeIsVisible);
-      this.adjustToggleButton(this.fileTreeIsVisible);
+      this.updateState(this.fileTreeIsVisible);
     },
-    adjustToggleButton(visible) {
+    updateState(visible) {
       const [toShow, toHide] = document.querySelectorAll('.diff-toggle-file-tree-button .icon');
-      toShow.classList.toggle('gt-hidden', visible);  // hide the toShow icon if the tree is visible
-      toHide.classList.toggle('gt-hidden', !visible); // similarly
-
-      const diffTree = document.getElementById('diff-file-tree');
-      diffTree.classList.toggle('gt-hidden', !visible);
+      const tree = document.getElementById('diff-file-tree');
+      if (visible) {
+        showElem(tree);
+        hideElem(toShow);
+        showElem(toHide);
+      } else {
+        hideElem(tree);
+        hideElem(toHide);
+        showElem(toShow);
+      }
```

Comment:
I was afraid of `toggleElem` because it had these runMode checks etc, but reading through it, it seems like that's something slated for future removal.

Reply:
Done